cmd/demo/README.md

```diff
@@ -0,0 +1,62 @@
+## About
+
+This folder contains system test demo for the ws connector, which is going to receive exported data from a local testnet
+
+### How it works
+
+These scripts provide a demo to help you configure and test your ws connector/driver to receive data from a local
+testnet. The scripts found in this `demo` folder:
+
+- `local-testnet.sh` creates a local testnet with one shard(0) + metachain. Each shard has 2 nodes(1 validator + 1
+  observer)
+- `observer-outport.sh` creates an observer in a specified shard(shard/metachain). This node acts an outport driver for
+  the ws connector. When started, it will sync from genesis and export data(blocks, validators info, etc.) to your ws
+  driver connector.
+
+Inside `cmd/connector` folder, there is a `main.go` file which represents the ws connector(data receiver).
+
+> **TIP**: Once you have experimented with this local setup, you might want to test the driver with exported data from
+> an observing squad(mainnet/devnet/testnet). [Here](https://github.com/multiversx/mx-chain-observing-squad) you can find
+> how to run your own observing squad. In order to enable nodes to export data, one needs to set
+> **[HostDriverConfig].Enabled =
+> true**  [from this config file](https://github.com/multiversx/mx-chain-go/blob/master/cmd/node/config/external.toml)
+
+## How to use
+
+1. Create and start a local testnet:
+
+```bash
+cd scripts
+./local-testnet.sh new
```

Comment:
`local-testnet` isn't executable. 2 options:
1. make it executable and commit it this way
2. extend the docs to run a `chmod +x local-testnet.sh` before

Reply:
Right, these are shell scripts.
I've added the `bash` prefix for the commands

Reply:
Right, these are shell scripts.
I've added the `bash` prefix for the commands